ssg/build_yaml.py

```diff
@@ -1427,7 +1418,31 @@ def to_xml_element(self):
             #
             # Additionally, we build the content (check subelement) here rather
             # than in xslt due to the nature of our SCE metadata.
-            check = ET.SubElement(rule, "check")
+            #
+            # Finally, before we begin, we might have an element with both SCE
+            # and OVAL. We have no way of knowing (right here) whether that is
+            # the case (due to a variety of issues, most notably, that linking
+            # hasn't yet occurred). So we must rely on the content author's
+            # good will, by annotating SCE content with a complex-check tag
+            # if necessary.
+
+            if 'complex-check' in self.sce_metadata:
+                # Here we have an issue: XCCDF allows EITHER one or more check
+                # elements OR a single complex-check. While we have an explicit
```

Comment:
I have added an `sce-bash.template` to the package_installed template. When I build a datastream I get 3 `check` elements within the `Rule` elements. First`check` is for OVAL, second `check` for SCE and a third `check` for OCIL. Is that expected? When will the `complex-check` element created?

Reply:
From my reading of the XCCDF spec, multiple top-level checks in different languages are implicitly joined with an "OR" and it is up to the implementation/execution environment to choose which to execute.

See the documentation section (already merged) on complex-check [for adding one](https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer/06_contributing_with_content.md#sce-check-content). Note that `complex-check` XML element requires the creation of the correct header (`# complex-check = <operator>`) for instance, and then the OCIL will be dropped into one side and then any OVAL/SCE content will be dropped into the other with the operator from the header.